README.md

```diff
@@ -27,6 +27,15 @@ Visual Studio downloads the required dependencies and compiles the Acrolinx .NET
 
 Refer to [sidebar demo] (https://github.com/acrolinx/acrolinx-sidebar-demo-dotnet/blob/master/README.md)
 
+## SDK Features
+
+1. **Document Model** - Provides [lookup](https://github.com/acrolinx/acrolinx-coding-guidance/blob/master/topics/text-lookup.md "Lookup") functionality.
```

Comment:
what's this "Lookup" inside the link? Shouln't it link to https://github.com/acrolinx/acrolinx-coding-guidance/blob/master/topics/lookup/change-tracking.md
What about the other lookup classes like the list of adapters or the lookup class?

Reply:
done